index.html

```diff
@@ -2,9 +2,45 @@
 <html>
   <head>
     <meta charset="utf-8">
-    <title></title>
+    <link
+      href="./style.css"
+      rel="stylesheet"
+      type="text/css">
+    <title>
+        "To-Do List"
+    </title>
   </head>
   <body>
-    <h1>"Hello World"</h1>
-  </body> 
+    <h1 class="titulo"> To-Do List </h1>
+    <div class="options">
+      <div class="newlist">
+        <span> Add New List </span>
+        <button> + </button>
+      </div>
+      <div class="clone">
+        <span> Clone </span>
+        <button> + </button>
+      </div>
+    </div>
+    <div class="list">
+      <header class="list-header">
+        <button class="newlistitem"> + </button>
```

Comment:
Add "-" dash to separate words in CSS class names

Reply:
Corrected in all lines